test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -9458,6 +9459,7 @@ def create_import_hook(self, name, attrs=None, body=None, overwrite=True):
                           Defaults to True
         :returns: True on success and False otherwise
         """
+        t = int(time.time()) - 20
```

Comment:
What is the reason for going 20 seconds in the past? Is it for accounting for differences in systems' times?

Reply:
Yes.  
I can add a comment above this, to make it clear.